### PR TITLE
UID2-2094 Add token response metric recording when cstg endpoint processing fails some checks

### PR DESCRIPTION
**Pull request by sunnywu**
…ls some checks

### CODE REVIEW

File: src/main/java/com/uid2/operator/vertx/UIDOperatorVerticle.java [hunk 1]
```diff
@@ -317,12 +318,13 @@ private void handleClientSideTokenGenerateImpl(RoutingContext rc) throws NoSuchA
             body = rc.body().asJsonObject();
         } catch (DecodeException ex) {
             ResponseUtil.Error(ResponseStatus.ClientError, 400, rc, "json payload is not valid");
+            recordTokenResponseStats(null, TokenResponseStatsCollector.Endpoint.ClientSideTokenGenerateV2, TokenResponseStatsCollector.ResponseStatus.BadJsonPayload);
```
Comment (jon8787) [inline src/main/java/com/uid2/operator/vertx/UIDOperatorVerticle.java:321]: can we make a method that calls both ResponseUtil.Error and recordTokenResponseStats? Seems like it's never correct to call the 1st without the 2nd

Comment (sunnywu) [reply src/main/java/com/uid2/operator/vertx/UIDOperatorVerticle.java:321]: there are 38 calls to com.uid2.operator.service.ResponseUtil#Error so i can create a new ticket about the proposed improvement but don't really wanna make this change in this small PR? 

There are quite a lot of instance where we don't call recordTokenResponseStats and should confirm the reason before making wholesale changes

Comment (jon8787) [reply src/main/java/com/uid2/operator/vertx/UIDOperatorVerticle.java:321]: Right but I'm not saying change 38 calls, just the new ones you've introduced. The 38 calls can be a future thing to investigate.

Comment (sunnywu) [reply src/main/java/com/uid2/operator/vertx/UIDOperatorVerticle.java]: @jon8787 done please have a look again

**Review by jon8787 — APPROVED** [review]

**Pull request merged**